ts/api/ProxyApi.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Proxy client using apisauce
+ */
+
+import apisauce, { ApiResponse } from "apisauce";
+
+import { apiDefaultTimeoutMs, apiUrlPrefix as proxyBaseURL } from "../config";
```

Comment:
why the rename of `apiUrlPrefix` ? renames are usually needed when there's a name clash, in this case there's no clash